pyproject.toml

```diff
@@ -181,6 +181,16 @@ pyiceberg = [
 postgis = [
     "psycopg2-binary>=2.9.1"
 ]
+workspace = [
+    "duckdb<1.2.1",
```

Comment:
is there a problem with newer version? we should be compatible with our duckdb extra. so the newest one should be allowed (I think it is >0.9)

Reply:
The newer versions have this iceberg bug, no? But I put it to >0.9 now.